tags/comments.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Comments
```

Comment:
Perhaps the title (and filename) of this page should be singular `Comment` to match the other pages in this section.